src/test/scala/mesosphere/marathon/integration/RemoteRegionOffersIntegrationTest.scala

```diff
@@ -0,0 +1,111 @@
+package mesosphere.marathon
+package integration
+
+import java.util.UUID
+
+import mesosphere.AkkaIntegrationTest
+import mesosphere.marathon.integration.facades.ITEnrichedTask
+import mesosphere.marathon.integration.facades.MarathonFacade.extractDeploymentIds
+import mesosphere.marathon.integration.facades.MesosFacade.ITResourceStringValue
+import mesosphere.marathon.integration.setup._
+import mesosphere.marathon.raml.App
+import mesosphere.marathon.state.{ FaultDomain, PathId, Region, Zone }
+import mesosphere.marathon.state.PathId._
+import mesosphere.mesos.Constraints
+import org.scalatest.Inside
+
+import scala.concurrent.duration._
+
+@IntegrationTest
+class RemoteRegionOffersIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest with Inside {
+
+  override lazy val mesosNumMasters = 1
+  override lazy val mesosNumSlaves = 3
+
+  class Fixture {
+    val homeRegion = Region("home_region")
+    val homeZone = Zone("home_zone")
+
+    val remoteRegion = Region("remote_region")
+    val remoteZone1 = Zone("remote_zone1")
+    val remoteZone2 = Zone("remote_zone2")
+  }
+
+  val fixture = new Fixture
```

Comment:
Please create the fixture in each test case. My idea is to encapsulate all fixtures.

Reply:
Could you please elaborate on that?

Reply:
I just saw that you do

```
"Launch an instance of the app in the default region if region is not specified" in new Fixture { ... }
```

So why is this still required? Because of 

```
override def mastersFaultDomains = Seq(Some(FaultDomain(region = fixture.homeRegion, zone = fixture.homeZone)))
```
?

Hm, this does not seems very clean.